public/index.html

```diff
@@ -82,8 +111,10 @@ <h2>PayJoin here to open these channels</h2>
 
 		</main>
 		<script type="text/javascript">
-			function add_channel() {
-				let channels = document.getElementById("channels");
+			let recommendations;
+			var channels = 1;
+			function add_channel(address=""){
```

Comment:
Doesn't add the new channel ID for me. Setting the `value` if `address != ""`:
```
function add_channel(address=""){
	var channels = document.getElementById("channels");
	let div = document.createElement('div');
	if (address == "") {
		div.innerHTML = `<input type="text" name="channels[${channels.length + 1}][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span="1" required><input type="number" name="channels[${channels.length + 1}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1" required>`;
	} else {
		div.innerHTML = `<input type="text" value="${address}" name="channels[${channels.length + 1}][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span="1" required><input type="number" name="channels[${channels.length + 1}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1" required>`;
	}
	channels.appendChild(div.firstChild);
	channels.appendChild(div.lastChild);
}
```

Reply:
Thanks, fixed in [b9790f0](https://github.com/chaincase-app/nolooking/pull/51/commits/b9790f0766e757564b6174a2935bdfb0667e3ad7)